editor/src/components/editor/actions/actions.tsx

```diff
@@ -1462,6 +1480,52 @@ function toastOnGeneratedElementsTargeted(
   return result
 }
 
+function toastOnUncopyableElementsSelected(
```

Comment:
what's the point of this function if it just calls `toastOnUncopyableElementsTargeted`?

Reply:
It was mirroring some existing code, as it seemed likely to be needed similarly where we can supply the paths being targeted separately.

Reply:
Removed in 9783c0bca.